src/mainboard/clevo/tgl-u/variants/ns50mu/devicetree.cb

```diff
@@ -344,15 +329,14 @@ chip soc/intel/tigerlake
 				device generic 0 on
 					chip drivers/intel/pmc_mux/conn
 						# TBT Type-C
-						register "usb2_port_number" = "6"
-						register "usb3_port_number" = "1"
+						register "usb2_port_number" = "6" # Should these be the TBT port, or the non TBT port?
```

Comment:
Of course the TBT port. It should match what is here: https://github.com/Dasharo/coreboot/pull/84/files#diff-03f56b06222463d9748ee4eacbe71e9517a7428ffb572446ab59d679749c5fa6R147 for USB3 port. The same goes for USB2 port automatically. These values are good, so you may remove the comments

Reply:
Thank you, removed the comment